pagerenderer/vue/core/src/main/java/com/peregrine/pagerender/vue/models/PageModel.java

```diff
@@ -320,7 +320,13 @@ public Tag(Resource r) {
             this.path = r.getPath();
             this.path = path.substring(path.indexOf("/jcr:content"));
             this.name = r.getName();
-            this.value = r.getValueMap().get("value", String.class);
+            String tag = r.getValueMap().get("value", String.class);
+            Resource tagResource = r.getResourceResolver().getResource(tag);
+            this.value = tagResource.getValueMap().get("value", String.class);
+
```

Comment:
if you delete a tag after you added the tag to the page this code will fail with a null pointer exception.